packages/crypto/src/identity.ts

```diff
@@ -1,4 +1,8 @@
-import { ZkIdentity, Strategy } from '@zk-kit/identity'
+import { BigNumber } from '@ethersproject/bignumber'
+import { randomBytes } from '@ethersproject/random'
+import _sha256 from 'crypto-js/sha256'
+import { poseidon } from 'circomlibjs'
+
 import { SnarkBigInt } from './crypto'
 
 type EddsaPrivateKey = Buffer
```

Comment:
I think we should store all types in one files(types.ts). Let do it in other PR!